Reply/app/src/main/java/com/example/reply/data/Email.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.reply.data
+
+import com.example.reply.data.local.LocalAccountsDataProvider
+
+/**
+ * A simple data class to represent an Email.
+ */
+data class Email(
+    val id: Long,
+    val sender: Account,
+    val recipients: List<Account> = emptyList(),
+    val subject: String = "",
+    val body: String = "",
```

Comment:
[nit] I generally found if I am making a variable have a default value of an empty string, then it should probably be a nullable type instead, with the default set to null. This helps having to avoid understanding what an empty string means - and avoids needing to do content checks to.

Reply:
Yes it shouldn't have default string values and non nullable as well because it's email's subject and body. removed empty strings.